generic/piranha-tree-sitter/src/cleanup_rules/java/edges.toml

```diff
@@ -0,0 +1,71 @@
+# The edges in this file specify the the flow between the rules
+
+[[edges]]
+from = "Returns boolean"
```

Comment:
thoughts between space and _? Should this be something like 
`returns_boolean`, `boolean_literal_cleanup`, etc?